app/views/helper/main.scala.html

```diff
@@ -12,14 +12,16 @@
         <title>@title - Short Format Pre-Sentence Report</title>
 
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta http-equiv="x-ua-compatible" content="IE=edge" >
```

Comment:
I thought we needed this to prevent IE from falling back to compatibility mode? Something to do with TPP machines (or the VM in SR2 - I can't quite remember) being set to compatibility  mode for "intranet" sites?

Reply:
There was something I read recently that talked about removing this if you're not targeting < IE11 but I've reverted this change just incase.